docs/0.6.0-alpha/stdlib/doc/env.md

```
@@ -0,0 +1,703 @@
+Prints a text with a specified color.
+
+### Usage
+```ab
+import { echo_colored } from "std/env"
+
+echo_colored("Red text", "red")
+echo_colored("Blue text", 34)
+```
+
+### Supported color names
+| Color name | Code |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)